tools/cst_python.py

```diff
@@ -367,6 +407,75 @@ def readCST(f):
 		if e['type'] == 'sndS':
 			e['content'] = f.read(fEntryLength)
 
+
+		if e['type'] == 'sndH':
+			# e['content'] = f.read(fEntryLength)
+
+			f.read(4) # unknown
+
+			soundLength = struct.unpack('>i', f.read(4) )[0]
+
+			f.read(4) # what
```

Comment:
i'm terrible at consistency, this is just garbage data and should use f.seek() instead